tools/build/config/rollup.config.mjs

```diff
@@ -159,6 +161,8 @@ const configForFormat = format => ({
     svgr(),
 
     terser(),
+
+    sizes(),
```

Comment:
We could probably use this and get rid of the current size analyser action we have

Reply:
The pro of the action is that it would alert us when something is too large